.github/workflows/ca-tests.yml

```diff
@@ -1196,3 +1196,124 @@ jobs:
+        run: |
+          docker network connect example client --alias client.example.com
+          docker inspect -f '{{ .NetworkSettings.Networks.example.IPAddress }}' client > client.ip
+          echo "Client's IP: `cat client.ip`"
+
+      - name: Install dependencies in PKI container
+        run: docker exec pki dnf install -y 389-ds-base
+
+      - name: Install DS in PKI container
+        run: docker exec pki ${PKIDIR}/tests/bin/ds-create.sh
+
+      - name: Install CA in PKI container
+        run: docker exec pki pkispawn -f /usr/share/pki/server/examples/installation/ca.cfg -s CA -v
+
+      # https://github.com/dogtagpki/pki/wiki/Configuring-SCEP-Responder
+      - name: Configure SCEP in CA
+        run: |
+          docker exec pki pki-server ca-config-set ca.scep.enable true
+          docker exec pki bash -c "echo UID:$(cat client.ip) > /etc/pki/pki-tomcat/ca/flatfile.txt"
+          docker exec pki bash -c "echo PWD:Secret.123 >> /etc/pki/pki-tomcat/ca/flatfile.txt"
+          docker exec pki pki-server restart --wait
+
+      # https://github.com/dogtagpki/pki/wiki/Building-SSCEP
+      - name: Build SSCEP in client container
+        run: |
+          docker exec client dnf install -y autoconf automake libtool openssl-devel rpm-build
+          docker exec client bash -c "mkdir -p ~/rpmbuild/SOURCES"
+          docker exec client curl -JLOSs https://github.com/certnanny/sscep/archive/v0.9.0/tags/sscep-0.9.0.tar.gz
+          docker exec client bash -c "mv sscep-0.9.0.tar.gz ~/rpmbuild/SOURCES"
+          docker exec client curl -JLOSs https://raw.githubusercontent.com/certnanny/sscep/v0.9.0/scripts/sscep.spec
+          docker exec client sed -i '/%build/ a ./bootstrap.sh' sscep.spec
+          docker exec client rpmbuild -ba sscep.spec
+          docker exec client bash -c "find ~/rpmbuild/RPMS -name *.rpm" > files
+          docker exec client dnf localinstall -y $(cat files)
+          docker exec client curl -JLOSs https://raw.githubusercontent.com/certnanny/sscep/v0.9.0/mkrequest
+          docker exec client chmod +x mkrequest
+
+      # https://github.com/dogtagpki/pki/wiki/Certificate-Enrollment-with-SSCEP
+      - name: Get CA certificate using SSCEP
+        run: |
+          docker exec client sscep getca \
+              -u http://pki.example.com:8080/ca/cgi-bin/pkiclient.exe \
+              -c ca.crt
+          docker exec client openssl x509 -text -noout -in ca.crt
+
+      - name: Enroll certificate with IP address using SSCEP
+        run: |
+          docker exec client ./mkrequest -ip $(cat client.ip) Secret.123
+          docker exec client openssl req -text -noout -in local.csr
+          docker exec client sscep enroll \
+              -u http://pki.example.com:8080/ca/cgi-bin/pkiclient.exe \
+              -c ca.crt \
+              -k local.key \
+              -r local.csr \
+              -l local.crt \
+              -E 3des \
+              -S sha256
```

Comment:
The last time I played with sscep client on an older platform it didn't support sha256 or 3des and I had to compile one separately.  Do you know if this sscep client will support them?

Reply:
Yes, it's documented here (look for the `-E` and `-S` params):
https://github.com/certnanny/sscep
If those params are not specified, by default it will use `des` and `md5`, and the CI will fail. Since the CI passed I suppose those params did work.

Reply:
Just FYI there's one unrelated failure in CI caused by a known JSS issue: https://github.com/dogtagpki/jss/issues/781